TelemetrySite/server/data_upload_scripts/data_upload.py

```diff
@@ -131,11 +144,13 @@ def format_data_bytes(bytes):
     return closeBracket
 
 
-## Main function to complete file conversion
-def main():
-    dotenv.load_dotenv("./credentials.env")
-    file_convert()
-
+# get the current progress of the file upload
+def get_progress(config_id):
+    if not config_id in progress_data:
+        return -1
```

Comment:
This function returns a failure or `None`. Did you mean to add more logic here?

Reply:
In this instance, no. If the key `config_id` isn't  in `progress_data`, then -1 is returned to stop the frontend from calling back. The frontend is setup to stop calling and end the progress bar if anything less than 0 is returned.